test/e2e_federation/federated-service.go

```diff
@@ -92,15 +92,15 @@ var _ = framework.KubeDescribe("Federated Services [Feature:Federation]", func()
 			clusters, primaryClusterName = getRegisteredClusters(UserAgentName, f)
 		})
 
-		Describe("service creation", func() {
+		Describe("Federated Service", func() {
```

Comment:
yes @madhusudancs, this PR is independent. However the 2 new testcase on services will fail with the current code.